runtime/gc_ctrl.c

```diff
@@ -171,6 +172,10 @@ CAMLprim value caml_gc_set(value v)
                      ARCH_INTNAT_PRINTF_FORMAT "u%%\n", caml_percent_free);
   }
 
+  newverb_gc = Long_val (Field (v, 3));
+
+  atomic_store_rel(&caml_verb_gc, newverb_gc);
```

Comment:
Wait, do you mean `atomic_store_rel(ease)` or `atomic_store_relaxed`? (Currently you are pairing store-releases with read-relaxed, which I'm not sure makes sense.)

Note: I find this naming choice confusing and I wish we used `_release` and `_acquire` instead of `_rel` and `_acq`.

Reply:
You are perfectly right, good catch.
I agree that the naming is quite confusing, but I wonder how doable it would be to do a full renaming of the codebase without annoying everyone too much.

Reply:
`release` is already abbreviated into `rel` in `memory_order_acq_rel` so that's a good reminder.